monitoring/defaults_linux.go

```diff
@@ -51,8 +51,8 @@ func PreInstallPortChecker() health.Checker {
 }
 
 // DefaultProcessChecker returns checker which will ensure no conflicting program is running
-func DefaultProcessChecker() health.Checker {
-	return &ProcessChecker{[]string{
+func DefaultProcessChecker(openEBSEnabled bool) health.Checker {
```

Comment:
I don't think satellite should be aware of OpenEBS specifically as it's intended to be a general purpose monitoring tool. What I would probably do, is extend it with a list of additional processes to check and pass `iscsid` from Gravity i.e. something like:

```go
func DefaultProcessChecker(additionalProcesses ...string) health.Checker { ... }
```

Reply:
Strongly agree - default presets do not work most of the time. I'm not sure how many of these are use in gravity, but I would definitely make the defaults exported (e.g. the list of processes) and add the ability to set the configuration externally so it would be possible to configure tests without touching the satellite checker code.